package.json

```diff
@@ -100,6 +100,7 @@
     "didStoreCluster": "npm:@ew-did-registry/did-ipfs-store@0.8.1-alpha.1140.0",
     "didStoreInfura": "npm:@ew-did-registry/did-ipfs-store@0.7.1-alpha.795.0",
     "dotenv": "^16.0.3",
+    "eth-ens-namehash": "^2.0.8",
```

Comment:
This package is used in ethers/utils.ts, but not installed. Compilation was sucessful, because package was provided by passport-did-auth, but when you install passport-did-auth locally, then compilation fails

Reply:
@JGiter ah, nice find. I had to manually fiddle with the files a bit to get to work but your explanation makes sense and this fix is better.

Reply:
I suppose this was an example of a "phantom" dependency. Perhaps this motivates the use of pnpm in the future: https://rushjs.io/pages/maintainer/package_managers/

Reply:
Yes, perhaps it will be more robast. There is also corresponding eslint rule https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md